src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -494,6 +573,12 @@ public CourseSummaryBundle getCourseSummaryWithoutStats(String courseId) throws
         return getCourseSummaryWithoutStats(cd);
     }
     
+    /**
+     * Returns a list of {@link CourseAttributes courseAttributes} for all courses a given student is enrolled in. 
```

Comment:
If the label (i.e. `courseAttributes`) is the same as the linked value (i.e. `CourseAttributes`), perhaps you could omit the label and just leave the linked value